src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -68,7 +68,7 @@ const convertEntityIdsToTags = (ids: string[], entities: CLM.EntityBase[]): OF.I
 }
 
 // Entities that can be chosen for required / blocking
-const conditionalEntityTags = (entities: CLM.EntityBase[]): IConditionalTag[] =>  {
+const conditionalEntityTags = (entities: CLM.EntityBase[]): IConditionalTag[] => {
```

Comment:
Hmm, don't know why this formatting change is showing up here. I thought I did a pass to format all the files to prevent this.  I must have missed some or maybe these got changed in later comments with the new entity stuff.

Reply:
I'm surprised as well given VSC autoformatting should have set this correct the first go 'round